src/components/Header.svelte

```diff
@@ -20,18 +20,20 @@
     <div class="header__menu--mobile" transition:fly="{{ y: 200, duration: 500 }}" >
       <ul class="header__menu">
         <li class="header__menu__item"><span data-smoothscroll="#vision">Vision</span></li>
-        <li class="header__menu__item"><span data-smoothscroll="#mission">Mission</span></li>
-        <li class="header__menu__item"><span data-smoothscroll="#purpose">Purpose</span></li>
-        <li class="header__menu__item"><span data-smoothscroll="#about-us">About us</span></li>
+        <li class="header__menu__item"><span data-smoothscroll="#why">Why</span></li>
+        <li class="header__menu__item"><span data-smoothscroll="#who-we-are">Who we are</span></li>
+        <li class="header__menu__item"><span data-smoothscroll="#needs">Needs</span></li>
+        <li class="header__menu__item"><span data-smoothscroll="#what-we-do">What we do</span></li>
       </ul>
```

Comment:
We'll talk about the menu (it might not be needed IMHO, if not to go to specific pages or with better namings)